amis/build_ami.sh

```diff
@@ -184,18 +184,15 @@ check_options() {
     esac
 
     # Ensure the specified architecture-OS combination is valid
-    if [ "${_arch}" == "arm64" ] && [[ "${_os}" =~ ^centos[6-7]$ ]]; then
+    if [ "${_arch}" == "arm64" ] && [[ "${_os}" =~ ^(centos7|alinux)$ ]]; then
```

Comment:
you are going to match also `alinux2` which I don't think is wanted

Reply:
Alinux2 should be excluded because there are `^` and `$` sorrounding characters, do you agree?

Reply:
you are right, I have missed the `$`